### PR TITLE
#164005396 login ui page

### PR DESCRIPTION
**Pull request by Nziranziza**
#### What does this PR do?
Add login page UI design
#### Description of Task to be completed?
- add html
- add css
- add responsive
#### How should this be manually tested?
- git clone the branch
- npm install
- npm run build-scss
- open ./ui/pages/auth.html
- click on login under the signup form and login login will flip to the front
#### Any background context you want to provide?
- [Desktop version Mockups](https://drive.google.com/drive/folders/1lcFZ3-tOAoN8g5_e6ORaKs8jEVI783jg)
- [Mobile version mockup](https://drive.google.com/drive/folders/1chupPda0Ny-iePagVr0_QCZgEhBabhM_)
#### What are the relevant pivotal tracker stories?
#164005396
###### Desktop version
screen shot 2019-02-25 at 19 05 15

###### Mobile version
screen shot 2019-02-25 at 19 05 56

#### Questions:
The login page will flip and this implies that to reach the login page you have to pass to the signup page. How will user access the login page without passing to the signup page???

### CODE REVIEW

**Review by oesukam — CHANGES_REQUESTED** [review]
@Nziranziza Try to make the space before and after social logos to be the same

File: ui/pages/auth.html [hunk 1]
```diff
@@ -51,9 +52,51 @@ <h2>SIGN UP</h2>
 							</div>
 						</div>
 						<div class="to-center" id="form-link">
-							<span>Already a member?</span><a href="#">Sign In</a>
+							<span>Already a member?</span><a onclick="flip()" href="#">Sign In</a>
```
Comment (oesukam) [inline ui/pages/auth.html:55]: @Nziranziza We are not supposed to use javascript for the mockup, just stick to html and css

File: ui/css/style.css [hunk 2]
```diff
@@ -1278,12 +1278,36 @@ input::placeholder {
   padding: 50px;
   border-radius: 10px;
   width: 450px;
-  max-width: 100vw !important;
-  margin: 0 15px; }
+  max-width: 100vw !important; }
```
Comment (oesukam) [inline ui/css/style.css]: @Nziranziza Try to avoid using !important

**Review by oesukam — CHANGES_REQUESTED** [review]
@Nziranziza There is only one main change that you can do that is to remove javascript as we are doing using html and css

File: ui/pages/auth.html [hunk 3]
```diff
@@ -51,9 +52,51 @@ <h2>SIGN UP</h2>
...
+								</div>
+							</div>
+							<div class="to-center" id="form-link">
+								<span>Not a member?</span><a onclick="flip1()" href="#">Sign Up</a>
+							</div>
+						</form>
+			</div>
+		</div>
+		<script type="text/javascript">
+			function flip(){
```
Comment (kimotho-njoki) [inline ui/pages/auth.html]: what happened to ES6?

File: ui/pages/auth.html [hunk 4]
```diff
@@ -51,9 +52,51 @@ <h2>SIGN UP</h2>
...
+			document.getElementById("card").style.transform = 'rotateY(180deg)';
+				console.log('hello')
```
Comment (kimotho-njoki) [inline ui/pages/auth.html]: Why are we console logging?

Comment (Nziranziza) [reply ui/pages/auth.html]: It was for debugging I had removed it

File: ui/pages/auth.html (hunk 3 above)
Comment (Nziranziza) [reply ui/pages/auth.html]: What do u mean???

**Review by oesukam — APPROVED** [review]
@Nziranziza Great work.

**Review by NkFab — APPROVED** [review]
Good work @Nziranziza

File: ui/pages/auth.html (hunk 3 above)
Comment (kimotho-njoki) [reply ui/pages/auth.html]: I meant why can't this function be written in ES6 format?

**Pull request merged**